openapi.yml

```diff
@@ -78,6 +78,23 @@ paths:
         <<: *common-responses
         200:
           description: Successful response
+  /organizations:
+    post:
+      security:
+        - bearerAuth: []
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/OrganizationRegistrationParams"
+      responses:
+        <<: *common-responses
+        200:
+          description: The registered organization
```

Comment:
As it's a Successful response, I'd say that the registered organization has been enacted upon...

If the organization is created, the response status code should be `201` and a `Location` header referencing the
created resource should be returned.